www/src/components/container.js

```diff
@@ -16,10 +16,12 @@ export default ({ children, className, hasSideBar = true, css = {} }) => (
       [presets.Tablet]: {
         paddingBottom: rhythm(1.5),
       },
-      ...css,
+      ...containerCss,
```

Comment:
Can you call this `overrideCSS`? As I think that's a slightly clearer description and matches what's happening elsewhere in the site.